Patch to allow bean-web to find stylesheet and images without requiring them to be in
the local directory.

diff -Nur beancount-1.0.orig/setup.py beancount-1.0/setup.py > beancount_web_fix.diff
--- beancount-1.0.orig/setup.py	2009-03-31 13:40:55.000000000 -0700
+++ beancount-1.0/setup.py	2010-09-12 14:54:36.521745100 -0700
@@ -63,6 +63,7 @@
                   'beancount.fallback',
                   'beancount.fallback.elementtree',
                   ],
+      package_data = {'beancount.web': ['*jpg', '*png', 'robots.txt', 'treetable.js', 'style.css']},
       scripts=scripts
      )
 
